### DISCUSSION TITLE
Shows great potential

### DISCUSSION THREAD

**Discussion by GougedEye** [EvaTheDM/mildly-magical-inspiration#17] [General]
So fired it up and managed to generate a sample deck. Really hope you are able to keep going with this as is shows great potential, even if it's not quite ready for primetime yet.

**Comment by EvaTheDM** [comment]
Hey! Thanks for the message! I’m definitely still working on the module. I’ve put most things on hold as I’ve realised it’s best to start making core improvements now before things get too complicated to deal with as such it might take still a bit until the next version is released but it’ll come with a lot of improvements!
